### PR TITLE
Defect/ffmt 3063: Multiple packing slip not working with Transfer

### PR DESCRIPTION
**Pull request by kibo-ajayjangam**

### CODE REVIEW

File: templates/back-office/transfer-packing-slip.hypr [hunk 1]
```diff
@@ -1,5 +1,86 @@
 ﻿{% extends "back-office/base" %}
 {% block body-content %}
+
+{% if model.packages %} 
+
+{% for packages in model.packages %}
```
Comment (PapaTaylor) [inline templates/back-office/transfer-packing-slip.hypr]: Please rename `packages` to its singular form, `package`

e.g.` {% for package in model.packages %}`

File: templates/back-office/transfer-packing-slip.hypr [hunk 2]
```diff
@@ -1,5 +1,86 @@
...
+                </tbody></table>
+                <table class="grid">
+                    <thead><tr>
+                        <th>Product Code</th>
+                        <th>Product Name</th>
+                        <th>Quantity</th>
+                        <th>Weight</th>
+                    </tr></thead>
+                    <tbody>
+                        {% for packagesItemDetails in packages.packingSlipItemDetails %}
```
Comment (PapaTaylor) [inline templates/back-office/transfer-packing-slip.hypr]: Please rename `packagesItemDetails` to be singular and remove trailing `s` from `packages`

e.g. `{% for packageItemDetail in package.packingSlipItemDetails %}`

File: templates/back-office/packing-slip-content.hypr [hunk 3]
```diff
@@ -98,7 +98,7 @@
                     <tbody>
                         {% for packagesItemDetails in packages.packingSlipItemDetails %}
```
Comment (PapaTaylor) [inline templates/back-office/packing-slip-content.hypr]: Please rename `packagesItemDetails` to be singular and remove trailing `s` from `packages`

e.g. `{% for packageItemDetail in package.packingSlipItemDetails %}`

File: templates/back-office/packing-slip-content.hypr [hunk 4]
```diff
@@ -98,7 +98,7 @@
...
                             {% for item in model.items %} <!-- model represents the shipment -->
-                                {% if packagesItemDetails.originalOrderItemId == item.originalOrderItemId %}
+                                {% if packagesItemDetails.lineId == item.lineId %}
```
Comment (PapaTaylor) [inline templates/back-office/packing-slip-content.hypr]: Change to `{% if packageItemDetail.lineId == item.lineId %}`

File: templates/back-office/transfer-packing-slip.hypr [hunk 5]
```diff
@@ -1,5 +1,86 @@
...
+                            {% for item in model.items %} <!-- model represents the shipment -->
+                                {% if packagesItemDetails.lineId == item.lineId %}
```
Comment (PapaTaylor) [inline templates/back-office/transfer-packing-slip.hypr]: Change to `{% if packageItemDetail.lineId == item.lineId %}`

File: templates/back-office/transfer-packing-slip.hypr [hunk 6]
```diff
@@ -1,5 +1,86 @@
...
+                                <tr>
+                                    <td>
+                                        {{ item.productCode }}
+                                    </td>
+                                    <td>
+                                        <div class="product-name">{{ item.name }}</div>
+                                    </td>
+                                    <td>
+                                        {{ packagesItemDetails.quantity }}
```
Comment (PapaTaylor) [inline templates/back-office/transfer-packing-slip.hypr]: Change to `{% if packageItemDetail.quantity %}`

**Review by PapaTaylor — CHANGES_REQUESTED** [review]
Looks good but please address requested changes.

**Review by kibo-kellytaylor — APPROVED** [review]
Looks good!

**Review by PapaTaylor — APPROVED** [review]
Looks good!

**Pull request merged**